src/main/java/frc/trigon/robot/utilities/SpeedMechanism2d.java

```diff
@@ -0,0 +1,114 @@
+package frc.trigon.robot.utilities;
+
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.Logger;
+
+public class SpeedMechanism2d {
+    private final String key;
+    private final Mechanism2d mechanism;
+    private final MechanismLigament2d
+            currentVelocityLigament,
+            currentVelocityTopArrowLigament,
+            currentVelocityBottomArrowLigament,
+            targetVelocityLigament,
+            targetVelocityTopArrowLigament,
+            targetVelocityBottomArrowLigament;
+    private static final Color8Bit
+            green = new Color8Bit(Color.kGreen),
+            red = new Color8Bit(Color.kRed),
+            blue = new Color8Bit(Color.kBlue);
+    private double deadband = 0.001;
+    private static final double
+            negativeTopAngle = 45,
+            negativeBottomAngle = 315,
+            positiveTopAngle = 210,
+            positiveBottomAngle = 145,
+            zeroTopAngle = 90,
+            zeroBottomAngle = 270;
```

Comment:
Static variables should be all caps and at the top of the class 
And the deadband should be final